druid/src/core.rs

```diff
@@ -114,6 +116,9 @@ pub(crate) struct WidgetState {
     pub(crate) children_changed: bool,
     /// Associate timers with widgets that requested them.
     pub(crate) timers: HashMap<TimerToken, WidgetId>,
+
+    // Port -> Host
+    pub(crate) sub_window_hosts: Option<Vec<WidgetId>>,
```

Comment:
Why not just vec? What's the difference between `None` and an empty vec?

Reply:
Good point